### DISCUSSION TITLE
Example for "opf"-File

### DISCUSSION THREAD

**Discussion by Gru0810** [advplyr/audiobookshelf#1101] [Q&A]
I try to migrate all my 3500 audiobooks to AudioBookShelf. I solved the problems with "Content" by using "desc.txt". I also solved the problem with narrator by using "reader.txt" . But I have problems to transfer "Series" an "episode-No". I didn't found an example of an "opf"-file which contains information about "series" and "episode-no". Please help me !

**Comment by advplyr** [comment]
You could use the recommended folder structure for series.

**Reply by Gru0810** [reply]
I tried it out and it worked fine.Thanks a lot for your great work !!
